web/packages/teleport/src/Clusters/Clusters.story.test.tsx

```diff
@@ -18,13 +18,14 @@
 
 import React from 'react';
 
-import { render, waitFor } from 'design/utils/testing';
+import { render, screen } from 'design/utils/testing';
 
 import { Story, createContext } from './Clusters.story';
 
 test('render clusters', async () => {
   const ctx = createContext();
   const { container } = render(<Story value={ctx} />);
-  await waitFor(() => document.querySelector('table'));
+
+  await screen.findByRole('table');
```

Comment:
Oh, this appears to be flaky now 🥲

Reply:
Ok, I think I know why.
When the table is rendered for the first time, we render unsorted rows.
Then, there is a `useEffect` in the table component that sorts them:

https://github.com/gravitational/teleport/blob/198521c9a668f508b89bf3d844ed6fbe09144cf9/web/packages/design/src/DataTable/useTable.ts#L174

Probably the best would be to get rid of it, but it's too much work. 

But I also noticed that when the table is rendered for the first time, it doesn't have the pagination header.
Maybe we could wait for it and when it appears, we could assume that the component has been fully rendered?